integration-cli/docker_cli_ps_test.go

```diff
@@ -554,3 +557,23 @@ func (s *DockerSuite) TestPsFormatHeaders(c *check.C) {
 		c.Fatalf(`Expected 'NAMES\ntest\n', got %v`, out)
 	}
 }
+
+func (s *DockerSuite) TestPsDefaultFormatAndQuiet(c *check.C) {
+	config := `{
+		"psFormat": "{{ .ID }} default"
+}`
+	d, err := ioutil.TempDir("", "integration-cli-")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(d)
+
+	err = ioutil.WriteFile(filepath.Join(d, "config.json"), []byte(config), 0644)
```

Comment:
geez its annoying to be put in this position where two flags control the same thing. sigh.

IMO, we shouldn't error because "no `--format`" == `--format="table"` to me.  I'd prefer to have "-q" do what it does today - meaning, overrides the default format.